### ISSUE TITLE
digital_ocean: private_networking broken with dopy==0.3.5

### ISSUE THREAD

**Issue by leucos** [ansible/ansible-modules-core#1412]
In the 0.3.5 version, dopy changed arguments order in `new_droplet`, inserting `ipv6` before `private_networking` (https://github.com/devo-ps/dopy/issues/27).

As a result, and since `digital_ocean`module is using plain arguments (https://github.com/ansible/ansible-modules-core/blob/devel/cloud/digital_ocean/digital_ocean.py#L242), it can not create a droplet with private networking enabled with dopy==0.3.5

While Dopy shouldn't change arguments order, Ansible's `digital_ocean` module should use named arguments to avoid this problem and increase robustness.

As a workaround, affected users can downgrade dopy (`sudo pip2 uninstall dopy; sudo pip2 install dopy==0.3.4`)

**Comment by leucos** [comment]
@bennojoy This is fixed upstream in dopy 0.3.6. May be a note should be added stating that dopy >= 0.3.2 (except 0.3.5) is required ?

**Comment by ansibot** [comment]
@alukovenko, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by alukovenko** [comment]
Going to get rid of dopy in #4219 
wontfix
